### DISCUSSION TITLE
Is there a discord server

### DISCUSSION THREAD

**Discussion by Kitaowo** [discord-jda/JDA#1592] [Questions and Help] [answered]
Is there a discord server where I can get a link?

**Comment by JellyBrick** [comment, marked as answer]
[discord-invite]: https://discord.gg/0hMr4ce0tIl3SLv5
[discord-shield]: https://discord.com/api/guilds/125227483518861312/widget.png
[ ![discord-shield][] ][discord-invite]

**Comment by Andre601** [comment]
The Discord is mentioned within the README.md file. You can't really miss it, since it's at the very top of it in form of a badge.
